CPAC/pipeline/engine.py

```diff
@@ -2053,8 +2094,18 @@ def ingress_raw_func_data(wf, rpool, cfg, data_paths, unique_id, part_id, ses_id
     func_wf.get_node("inputnode").iterables = ("scan", list(func_paths_dct.keys()))
 
     rpool.set_data("subject", func_wf, "outputspec.subject", {}, "", "func_ingress")
-    rpool.set_data("bold", func_wf, "outputspec.rest", {}, "", "func_ingress")
+    reorient = pe.Node(
+        interface=afni.Resample(),
+        name=f"reorient_func_{part_id}_{ses_id}",
+    )
+    reorient.inputs.orientation = cfg.pipeline_setup["desired_orientation"]
+    reorient.inputs.outputtype = "NIFTI_GZ"
+    wf.connect(func_wf, "outputspec.rest", reorient, "in_file")
+    rpool.set_data("bold", reorient, "out_file", {}, "", "func_ingress")
+    # rpool.set_data("bold", func_wf, "outputspec.rest", {}, "", "func_ingress")
+
```

Comment:
If we do this, then this can just be
```suggestion
    reorient = rpool.reorient_node("func", "func_ingress", f"{part_id}_{ses_id}")
    wf.connect(func_wf, "outputspec.rest", reorient, "in_file")
```

Reply:
with the changes, I think this is not needed.